src/jdk.accessibility/windows/classes/com/sun/java/accessibility/internal/AccessBridge.java

```diff
@@ -164,7 +166,7 @@ public final class AccessBridge {
     private static void initStatic() {
         // Load the appropriate DLLs
         boolean is32on64 = false;
-        if (System.getProperty("os.arch").equals("x86")) {
+        if (Architecture.isX86()) {
```

Comment:
This would be another coupling of java.desktop into java.base internals and I don't think it worth it.
Particularly because this check is obsolete !
It is a remnant of when accessbridge was an unbundled download so provided both 32 and 64 bit.
We don't even have  the file it is looking for in the build - even if someone did a 32 bit build.
Instead of making this change a bug should be filed to remove this code.

Reply:
Reverting; filed [8305745](https://bugs.openjdk.org/browse/JDK-8305745)